.github/workflows/ci.yaml

```diff
@@ -0,0 +1,37 @@
+name: CI
+on:
+  workflow_dispatch: # Manual invocation.
+  push:
+jobs:
+  CI:
+    runs-on: ubuntu-latest
+    permissions:
+      # required by aws-actions/configure-aws-credentials
+      id-token: write
+      contents: read
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup Java
+        uses: actions/setup-java@v3
+        with:
+          java-version: '8'
+          distribution: 'corretto'
+          cache: 'sbt'
+      - name: SBT
+        run: sbt clean compile test assembly
+      - uses: aws-actions/configure-aws-credentials@v3
```

Comment:
there is a v4 of this - not sure if we are using it more widely?

[aws-actions/configure-aws-credentials@](https://github.com/aws-actions/configure-aws-credentials#v4-announcement-91123)